docs/source/operations/truenas-storage-for-zero-cluster.md

```diff
@@ -0,0 +1,645 @@
+# TrueNAS NFS Storage for Zero OpenShift Cluster
+
+## Overview
+
+We repurposed an recently retired Fujitsu server, `10.0.17.8`, to
+use as temporary storage to relieve storage capacity issues on the
+[Zero cluster][].
+
+[zero cluster]: https://console-openshift-console.apps.zero.massopen.cloud/
+
+### Connecting to the system
+
+You can reach the system by first logging into
+kzn-rtr.infra.massopen.cloud (or any of the OpenShift nodes).
+
+You can access the web UI at <http://10.253.0.10/>, or by setting up
+port forwarding from your local machine, e.g:
+
+```
+ssh -L 8080:10.253.0.10:80 kzn-rtr.infra.massopen.cloud
+```
+
+And then connecting to <http://localhost:8080>.
+
+You can log into the system using `ssh` if your ssh key has been added
+to `root`'s `authorized_keys` file, or you can connect as the `csi`
+user using the private key embedded in the OpenShift cluster:
+
+```
+oc -n democratic-csi get secret moc-nfs-democratic-csi-driver-config \
+  -o yaml |
+  yq -r '.data."driver-config-file.yaml"' |
+  base64 -d |
+  yq -r .sshConnection.privateKey
+```
+
+## Operating System
+
+The server runs [TrueNAS][] (neè [FreeNAS][]) 12, which is based on FreeBSD.
+
+[TrueNAS]: https://www.truenas.com/
+[FreeNAS]: https://www.freenas.org/
+
+## Networking
+
+One of the system's two 10Gb interfaces is attached to VLAN 210:
+
+```
+# ifconfig ix0
+ix0: flags=8943<UP,BROADCAST,RUNNING,PROMISC,SIMPLEX,MULTICAST> metric 0 mtu 1500
+        description: ix0
+        options=a538b9<RXCSUM,VLAN_MTU,VLAN_HWTAGGING,JUMBO_MTU,VLAN_HWCSUM,WOL_UCAST,WOL_MCAST,WOL_MAGIC,VLAN_HWFILTER,VLAN_HWTSO,RXCSUM_IPV6>
+        ether 90:1b:0e:0f:5d:8a
+        inet 10.253.0.10 netmask 0xfffffe00 broadcast 10.253.1.255
+        media: Ethernet autoselect (10Gbase-Twinax <full-duplex,rxpause,txpause>)
+        status: active
+        nd6 options=9<PERFORMNUD,IFDISABLED>
+```
+
+## Available drives
+
+The system has 39 838GB drives.
+
+```
+Geom name: da0
+Providers:
+1. Name: da0
+   Mediasize: 899627876352 (838G)
+   Sectorsize: 512
+   Mode: r1w1e2
+   descr: LSI MR SAS 6G 1GB
+   lunid: 6003005701e387502117a99e129b7f83
+   ident: 00837f9b129ea917215087e301570003
+   rotationrate: unknown
+   fwsectors: 63
+   fwheads: 255
```

Comment:
what is the importance of having these specific bits of information (ident, lunid, fwsecorts etc) about all these drives? I don't see any useful information here.